### PR TITLE
added caching config of pnpm-store for travisci

### PR DESCRIPTION
**Pull request by rstiller**

### CODE REVIEW

File: docs/continuous-integration.md [hunk 1]
```diff
@@ -12,8 +12,11 @@ On [Travis CI](https://travis-ci.org/), you can use pnpm for installing your dep
 ```yaml
 cache:
   npm: false
+  directories:
+    - "~/.pnpm-store"
 before_install:
   - curl -L https://unpkg.com/@pnpm/self-installer | node
+  - npm config set store-dir ~/.pnpm-store
```
Comment (zkochan) [inline docs/continuous-integration.md]: but that is the default location of the store. Why explicitly setting it?

Comment (rstiller) [reply docs/continuous-integration.md]: Just to remind people, who specify a custom location, to properly set it as well.

Comment (zkochan) [reply docs/continuous-integration.md]: ok, use `pnpm config` instead of `npm config`

Comment (rstiller) [reply docs/continuous-integration.md]: Done

**Pull request merged**